### PR TITLE
Add siva-head tool

### PR DESCRIPTION
**Pull request by vmarkovtsev**
As requested several times by non-Siva-native users.

### CODE REVIEW

File: PublicGitArchive/examples/siva-heads.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	}
+	sivaStorage := filesystem.NewStorage(fs, cache.NewObjectLRUDefault())
+	if sivaStorage == nil {
+		return nil, fmt.Errorf("unable to create a new storage backend for Siva file %s", path)
+	}
+	repo, err := git.Open(sivaStorage, tmpFs)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to open the Git repository from Siva file %s", path)
+	}
+	log.Print("Reading the references, this make take some time... ")
```
Comment (jfontan) [inline PublicGitArchive/examples/siva-heads.go]:
```suggestion
	log.Print("Reading the references, this may take some time... ")
```

**Comment by jfontan** [comment]
Travis information is stuck. The tests passed :tada:

**Comment by vmarkovtsev** [comment]
@jfontan Shall I merge?

**Comment by jfontan** [comment]
@vmarkovtsev done

**Pull request merged**